tests/features/common/kieserver/kieserver-common.feature

```diff
@@ -29,4 +29,16 @@ Feature: Kie Server common features
       | KIE_SERVER_MGMT_DISABLED    | true     |
       | KIE_SERVER_STARTUP_STRATEGY | invalid  |
     Then container log should contain -Dorg.kie.server.mgmt.api.disabled=true
-    And container log should contain The startup strategy invalid is not valid, the valid strategies are LocalContainersStartupStrategy and ControllerBasedStartupStrategy
\ No newline at end of file
+    And container log should contain The startup strategy invalid is not valid, the valid strategies are LocalContainersStartupStrategy and ControllerBasedStartupStrategy
+  
+  Scenario: Don't configure kie server to use LDAP authentication
```

Comment:
Is this env enough to know if the user provided all needed envs to setup the Ldap login module correctly?

Reply:
No, but there are so many different combinations that I didn't want to make the logic too complex. For other integrations like SSO, the URL is usually the main discriminator.